main.go

```diff
@@ -233,11 +293,12 @@ func main() {
 		log.Fatal("Must provide an API key!")
 	}
 
-	if shouldEncrypt == nil && shouldDecrypt == nil && shouldValidateCiphertext == nil {
+	if shouldEncrypt == nil && shouldDecrypt == nil && shouldValidateCiphertext == nil && shouldSign == nil && shouldVerify == nil {
 		log.Fatal("Must specify either encrypt OR decrypt OR is-peacemakr-ciphertext")
```

Comment:
maybe want it to say "must select an action, encrypt, decyprt, is-peacemakr-ciphertext, sign, verify.."

Also, is a signed block going to be true for a call to `is-peacemakr-ciphertext`?

Reply:
yup it works :)

Reply:
hmmm... At some point we may want to change the language around the predicate to `is-peacemakr-blob` since a signature is not a ciphertext?

Reply:
or even add in a separate one that is `is-peacemakr-signature` for the other one, if we can detect that?

Reply:
i like the idea of renaming it to is-peacemakr-blob